.flake8

```diff
@@ -13,5 +13,6 @@ exclude =
     build,
     .nox,
     venv,
-    .venv
+    .venv,
+    mqt/qecc/cc-decoder/__init__.py
```

Comment:
Why is this file excluded?
Doesn't matter too much anyway, as flake8 will be replaced by ruff in the foreseeable future anyway, but still.

Reply:
You can get rid of those by defining the __all__ property in the unit file. Ruff actually suggests that.
See the main __init__.py file.

Reply:
Just to bring it up again: this has not been fixed yet.

Reply:
:+1: